apps/web/components/shared/web3-feature-card.tsx

```
@@ -0,0 +1,60 @@
+  stats,
+  checkList,
+}: Web3FeatureCardProps) => (
+  <motion.div
+    initial={{ opacity: 0, y: 20 }}
+    whileInView={{ opacity: 1, y: 0 }}
+    viewport={{ once: true }}
+    transition={{ duration: 0.5, type: "spring" }}
+    whileHover={{ y: -5 }}
+    whileFocus={{ y: -5 }}
+    className="bg-white rounded-2xl p-8 shadow-sm hover:shadow-md transition-all duration-300"
+    role="article"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<article>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)